x-pack/test/functional/page_objects/ingest_pipelines_page.ts

```diff
@@ -141,7 +143,10 @@ export function IngestPipelinesPageProvider({ getService, getPageObjects }: FtrP
       if (maxmind) {
         await testSubjects.setValue('maxmindField', maxmind);
       }
-      await testSubjects.setValue('databaseNameSelect', databaseName);
+
+      await testSubjects.click('databaseNameSelect');
+      await pageObjects.common.sleep(1000);
+      await testSubjects.click(databaseName);
```

Comment:
I must say I'd prefer to see a click, followed by an assertion of what the click caused, instead of sleep.  
Especially as the needed sleep duration could be variable.

That way the test is less flaky.

Wdyt?

Reply:
Thanks for the review @wayneseymour! You were right that there is already a `selectValue` helper (not sure why I missed it) and it works locally! I added it in [fbcbed1](https://github.com/elastic/kibana/pull/196544/commits/fbcbed1d67dcb022f38a678de9a1bfab8c13bd15) and I will wait to see if it will also work in ci and with the flaky test runner.

I also replaced some of the sleep calls with retry, let me know what you think about this approach.